wcc-contentful/lib/wcc/contentful/middleware/store/caching_middleware.rb

```diff
@@ -62,20 +65,29 @@ def index?
 
     private
 
+    LAZILY_CACHEABLE_TYPES = %w[
+      Entry
+      Asset
+      DeletedEntry
+      DeletedAsset
+    ].freeze
+
     def _index(json)
       ensure_hash(json)
       id = json.dig('sys', 'id')
-      return unless prev = @cache.read(id)
+      type = json.dig('sys', 'type')
+      prev = @cache.read(id)
+      return if prev.nil? && LAZILY_CACHEABLE_TYPES.include?(type)
```

Comment:
This right here - at some point we stopped storing the sync token in the rails cache.  So it was doing a full resync on every restart, as well as every rake task in the heroku build.

Reply:
I wonder if this will help the initial page load time in any app, too!